token-plugins-guide.md

```diff
@@ -0,0 +1,180 @@
+# 1inch Token Plugins: A Comprehensive Guide for Extending ERC20 Functionalities
+
+[Overview](#overview)
+
+[Primary Benefits](#primary-benefits)
+
+[Implementation](#implementation)
+
+[Overview of Contract Structure](#overview-of-contract-structure)
+   - [ERC20Plugins.sol (Abstract Plugin-enabled ERC20 contract)](#erc20pluginssol-plugin-enabled-erc20-token-contract)
+   - [Plugin.sol (Abstract Plugin contract)](#pluginsol-abstract-plugin-contract)
+
+[Deployed Examples](#deployed-examples)
+
+[Helpful Links](#helpful-links)
+
+## Overview
+
+Token plugins are smart contracts that extend the capabilities of existing ERC20 tokens. They allow for additional accounting to be added to an existing token without the need to deposit your tokens into a contract. **_This is a revolutionary step in the evolution of DeFi_**, as the entire ecosystem has historically relied upon the transfer of tokens in and out of external contracts to achieve enhanced capabilities! 
```

Comment:
Those plugins can’t extend existing tokens, but I think we can say they extend plugin-enabled tokens or token-enabled token wrappers (ERC20Wrapper extension).

Reply:
Fixed